CMakeLists.txt

```diff
@@ -6,8 +6,9 @@ if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 14)
 endif()
 
+# Disable Wredundant-decls warnings since rosidl generates redundant function declarations
```

Comment:
Is there a corresponding bug report at rosidl?

Reply:
Filed up a [bug issue](https://github.com/ros2/rosidl/issues/421) at rosidl.